cdk/appsync/pools/schema.graphql

```diff
@@ -1,13 +1,18 @@
 ﻿type Pool
   @key(
-    name: "byTotalLiquidity",
```

Comment:
Didn't realize you could drop the commas. This definitely works?

Reply:
I just ran format command and commas were deleted, and yeah, it works ok